src/index.js

```diff
@@ -11,6 +11,7 @@ import reportWebVitals from './reportWebVitals';
 ReactDOM.render(
   <React.StrictMode>
     <Header />
+    <Login />
     <App />
     <Form />
     <Profile />
```

Comment:
Add the **Header**, **Login**, **Form**, **Profile** and **Footer** into your App and not in the index